### ISSUE TITLE
Selectable Text not working as expected

### ISSUE THREAD

**Issue by pmafpinho** [meliorence/react-native-render-html#634]
### Decision Table

- [X] My issue does not look like “The HTML attribute 'xxx' is ignored” (unless we claim support for it)
- [X] My issue does not look like “The HTML element `<yyy>` is not rendered”

### Good Faith Declaration

- [X] I have read the HELP document here: https://git.io/JBi6R
- [X] I have read the CONTRIBUTING document here: https://git.io/JJ0Pg
- [X] I have confirmed that this bug has not been reported yet

### Description

I'm trying to select text and when we use scrollview and pass with the finger, words are selected.

`<RenderHTMLConfigProvider defaultTextProps={{ selectable: true }} {...config}>
        {content}
      </RenderHTMLConfigProvider>`
      
      How can I fix it? I think a bug with scrollview?
      
     

### React Native Information

```sh
No make any sense to have this, because I can't debug this problem.
```

### RNRH Version

"react-native-render-html": "^6.3.4",

### Tested Platforms

- [X] Android
- [X] iOS
- [ ] Web
- [ ] MacOS
- [ ] Windows

### Reproduction Platforms

- [X] Android
- [X] iOS
- [ ] Web
- [ ] MacOS
- [ ] Windows

### Minimal, Reproducible Example

I don't have it on snak.

**Comment by jsamr** [comment]
@pmafpinho I did not understand the issue, can you rephrase or provide an example? Eventually, a video.

**Comment by NTienAnh** [comment]
> @pmafpinho I did not understand the issue, can you rephrase or provide an example? Eventually, a video.

Example like this
https://snack.expo.dev/0_Ifj62U-?platform=ios

**Comment by jsamr** [comment]
@NTienAnh 

If I understand it well by the in-code comment, your issue seems unrelated to OP's, who mentioned scrolling interactions. https://snack.expo.dev/@jsamr/text-fails-to-be-selectable-on-ios

> I'm trying to select text and when we use scrollview and pass with the finger, words are selected.

@pmafpinho If I got this right, some words are immediately selected while a user is performing a scrolling gesture, while we would expect instead that a selection event only happens when the user long-presses some text.

Native interactions like this are handled by React Native. This library does not perform anything else than rendering React Native components. The challenge is to manage to reproduce *without this library*, in vanilla RN, so that it can be reported to the RN repository. I don't have the bandwidth to investigate further what is likely a React Native bug, and without further reproduction, I will close this ticket in one week.

**Comment by pmafpinho** [comment]
Hi @jsamr and @NTienAnh yes, the problem is: when you scroll on a text, automatically words selection happens (android and iOS). I think it can be a react native bug? because I tested <Text select="true">Example</> and happens the same. You can close the issue.

**Comment by jsamr** [comment]
> I think it can be a react native bug?

@pmafpinho Yes definitely. What I suggest would be, firstly, manage to reproduce the smallest repro possible, and then investigate with Flipper to see the *native* tree of nodes and their props so that you're able to write down a vanilla reproduction and report to RN repository.

Atenciosamente, eu vou fechar agora.
